Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1
+
+# Build
+FROM golang:1.20-alpine AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY *.go ./
+
+RUN go build -o /img-LinuxFr.org
+
+RUN go install golang.org/x/vuln/cmd/govulncheck@latest
+RUN govulncheck /img-LinuxFr.org
```

Comment:
Nice tool :)

Reply:
now checking source and binary (probably same results expected)